### PR TITLE
Create pu-paris-nanterre.csl

### PR DESCRIPTION
**Pull request by ajustine**

### CODE REVIEW

File: pu-paris-nanterre.csl [hunk 1]
```diff
@@ -0,0 +1,598 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" default-locale="fr-FR" version="1.0" page-range-format="expanded" demote-non-dropping-particle="display-and-sort">
+  <info>
+    <title>Presses universitaires de Paris Nanterre (note, French)</title>
+    <title-short>pu_paris_nanterre</title-short>
+    <id>http://www.zotero.org/styles/PUPN</id>
```
Comment (libora) [inline pu-paris-nanterre.csl]: change "PUPN" to "pu-paris-nanterre"

File: pu-paris-nanterre.csl [hunk 2]
```diff
@@ -0,0 +1,598 @@
...
+    <link href="http://www.zotero.org/styles/PUPN" rel="self"/>
```
Comment (libora) [inline pu-paris-nanterre.csl]: change "PUPN" to "pu-paris-nanterre"

File: pu-paris-nanterre.csl [hunk 3]
```diff
@@ -0,0 +1,598 @@
...
+	  <text value="s.n."/>
+	  </else>
+	</choose>  
+      </if>
+      <else-if type="thesis" match="any">
+	<text value="(dactyl.)"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="yearpage">
```
Comment (libora) [inline pu-paris-nanterre.csl]: this macro is not used in style!!!

File: pu-paris-nanterre.csl [hunk 4]
```diff
@@ -0,0 +1,598 @@
...
+      </if>
+      <else>
+        <text variable="issue"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="collection">
+<text variable="collection-title" quotes="true" prefix="" suffix=""/>
+  </macro>
+<citation et-al-min="4">
```
Comment (libora) [inline pu-paris-nanterre.csl]: here must be defined  "et-al-use-firs" too

File: presses-universitaires-de-paris-nanterre.csl [hunk 5]
```diff
@@ -0,0 +1,592 @@
...
+          <text value="consulté le" suffix=" " prefix=" "/>
+          <date variable="accessed" form="text">
+            <date-part name="day"/>
+            <date-part name="month"/>
+            <date-part name="year"/>
+          </date>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
```
Comment (rmzelle) [inline presses-universitaires-de-paris-nanterre.csl]: @ajustine, repository styles are not allowed to contain any unused macros, so the entire "yeardate" macro should be deleted. (lines 212-312)

File: presses-universitaires-de-paris-nanterre.csl [hunk 6]
```diff
@@ -0,0 +1,592 @@
...
+        <single>dans</single>
+        <multiple>dans</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <choose>
+      <if variable="author">
+        <names variable="author">
+          <name form="long" sort-separator=" " delimiter=", " and="text" delimiter-precedes-last="never" delimiter-precedes-et-al="never" et-al-min="4" et-al-use-first="4" name-as-sort-order="all" font-style="normal">
```
Comment (rmzelle) [inline presses-universitaires-de-paris-nanterre.csl]: @ajustine, `et-al-min` must always have a higher value than `et-al-use-first`. (`et-al-min` specifies the minimum number of names in a name list that is required to activate et-al abbreviation, and `et-al-use-first` is the number of names shown in an abbreviated name list before "et al.")

File: presses-universitaires-de-paris-nanterre.csl [hunk 7]
```diff
@@ -0,0 +1,592 @@
...
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name form="long" and="text" delimiter-precedes-last="never" sort-separator=" " font-style="normal"></name>
+      <label form="short" prefix="&#160;(" suffix=".)"/>
+    </names>
+  </macro>
+  <macro name="container-author">
+    <names variable="container-author">
+      <name form="long" sort-separator=" " delimiter=", " and="text" delimiter-precedes-last="never" delimiter-precedes-et-al="never" et-al-min="4" et-al-use-first="4" name-as-sort-order="all" font-style="normal">
```
Comment (rmzelle) [inline presses-universitaires-de-paris-nanterre.csl]: @ajustine, same here, et-al-min must always have a higher value than et-al-use-first.

File: presses-universitaires-de-paris-nanterre.csl [hunk 8]
```diff
@@ -0,0 +1,592 @@
...
+      </if>
+      <else>
+        <text variable="issue"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="collection">
+    <text variable="collection-title" quotes="true" prefix="" suffix=""/>
+  </macro>
+  <citation et-al-min="4">
```
Comment (rmzelle) [inline presses-universitaires-de-paris-nanterre.csl]: @ajustine, it's clearer to always use `et-al-min` by `et-al-use-first` together on the same element.

File: presses-universitaires-de-paris-nanterre.csl [hunk 9]
```diff
@@ -0,0 +1,592 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" default-locale="fr-FR" version="1.0" page-range-format="expanded" demote-non-dropping-particle="display-and-sort">
+  <info>
+    <title>Presses universitaires de Paris Nanterre (note, French)</title>
+    <title-short>PU Paris Nanterre</title-short>
+    <id>http://www.zotero.org/styles/presses-universitaires-de-paris-nanterre</id>
+    <link href="http://www.zotero.org/styles/presses-universitaires-de-paris-nanterre" rel="self"/>
+    <link href="http://scd.u-paris10.fr/services/boite-a-outils-pour-la-recherche-/ressources-pratiques-591002.kjsp?RH=1390906967545" rel="documentation"/>
+    <link href="https://presses.u-paris10.fr/wp-content/uploads/2017/02/ConsignesAuteurs.pdf" rel="documentation"/>
```
Comment (rmzelle) [inline presses-universitaires-de-paris-nanterre.csl]: @ajustine, we'd also like to see a "template" link, assuming that you based this CSL style on an existing one. (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link)

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**